### ISSUE TITLE
Users cant install Brave Browser and Extensions in Brave in countries where Brave.com is blocked by Govt/ISP

### ISSUE THREAD

**Issue by NaaeemMalik** [brave/brave-browser#26274]
## Description 

brave.com is blocked in [Pakistan](https://www.reddit.com/r/brave_browser/comments/f141pd/why_is_bravecom_blocked_in_pakistan/) by dictators to not allow freedom of speech or anything, and it might be blocked in some other countries as well, but those people can open github.com, and not even people on reddit know that it can be downloaded from https://github.com/brave/brave-browser/releases/ and they are suggesting third party websites which can have malware in them.
so should I mention that link in [downloads section](https://github.com/NaaeemMalik/brave-browser/blob/master/README.md#downloads)?

people here are also unable to install extensions in Brave since I think brave needs to make connection to brave.com to get proxy data but it cant receive that so it says extension failed to install so to fix that I think we can make brave use no proxy if it cant access brave.com and this check is made every time when brave is started so if user changes ISP or country he can access it using proxy again. also nor Brave News neither any other feature works that depends on brave.com .

Brave Browser automatic Updates work thought. so it updates version number whenever updates are released.

## Steps to Reproduce

   1.  set a system proxy that blocks brave.com and related sites 
   2.  search Brave on google, like most people do, you are unable to open brave.com to install browser.
   3.  open Brave, try to install chrome extensions and you wont be able to. unless you download extension from other browser and install in Brave or maybe like reddit user suggested use a VPN

## Expected result:
user can install brave if he opens GitHub repo and install extensions without VPN or downloading from other browser

## Reproduces how often: 
big problem.

## Brave version (brave://version info)
all versions I have tried since 2 years. 
currently on Version 1.44.112 Chromium: 106.0.5249.119 (Official Build) (64-bit)

## Version/Channel Information:
all channels probably

- Can you reproduce this issue with the current release? YES
- Can you reproduce this issue with the beta channel?  Not Sure
- Can you reproduce this issue with the nightly channel? Not Sure

## Other Additional Information:

- Is the issue reproducible on the latest version of Chrome? NO, these features work in chrome and other browsers.

**Comment by NaaeemMalik** [comment]
@bsclifton I don't know who to tag here so I'm doing you

**Comment by pes10k** [comment]
I think thats a fine idea, as long as we link to the corresponding release in github (i.e., we dont want someone trying to download stable downloading the nightly release off github).

Including @diracdeltas too, since we've had discussions on reducing the number of places you can download brave from, to make it easier to know which sources are legit and trusted, and which are possibly malicious

**Comment by NaaeemMalik** [comment]
yes, we can do that, if you guys are maintaining a version on your website, you can maintain one on github as well probably :) 

we also need to figure out chrome extension support as well bcz after installing Brave people would still not be able to download extensions bcz your VPN doesnt work as well

**Comment by NaaeemMalik** [comment]
if we only change version in this url we can probably keep one url for every platform
 https://github.com/brave/brave-browser/releases/download/v1.46.144

**Comment by NaaeemMalik** [comment]
@BrendanEich @rebron @mattmcalister @bridiver

**Comment by bsclifton** [comment]
I think the https://versions.brave.com/ page can be used - maybe someone can mirror that
